src/parser/getConfig.ts

```diff
@@ -1,16 +1,28 @@
 import Expansion from 'game/Expansion';
 import AVAILABLE_CONFIGS from 'parser';
 
-export default function getConfig(expansion: Expansion, specId: number, type: string) {
+export default function getConfig(
+  expansion: Expansion,
+  specId: number,
+  type: string,
+  icon?: string,
+) {
   const expansionConfigs = AVAILABLE_CONFIGS.filter((config) => config.expansion === expansion);
   let config = specId !== 0 && expansionConfigs.find((config) => config.spec.id === specId);
+  // Classic
   if (!config) {
     config = expansionConfigs.find((config) => config.spec.id === 0 && config.spec.type === type);
+    if (!config && icon) {
+      config = expansionConfigs.find(
+        (config) => config.spec.type === type && config.spec.icon === icon,
```

Comment:
not a fan of searching by icon here. we should figure out what we want to assign for spec ids (WCL ids? other ids?) and use those instead of relying on the icon matching

Reply:
I'd like to match by spec id but WCLs has zero for every Class spec Id. Is there a way to get them to change that?

Reply:
Yes, we have spec IDs internally but the specID in combatantinfo is from blizzard. we could potentially override that for classic. i'll ask kihra about it.

Reply:
@emallson - per our discussion that icon is the most convenient way to do this